doc/advanced.rst

```diff
@@ -168,13 +178,38 @@ package in a module called ``scraper``. Here is the scraper code::
 Then, in our ``conf.py`` file, we include the following code::
 
    from mymodule import PNGScraper
-   my_scraper_instance = PNGScraper()
 
    sphinx_gallery_conf = {
        ...
-       'image_scrapers': ('matplotlib', my_scraper_instance),
+       'image_scrapers': ('matplotlib', PNGScraper()),
    }
 
+Example 3: matplotlib with SVG format
+-------------------------------------
+The :func:`sphinx_gallery.scrapers.matplotlib_scraper` supports ``**kwargs``
+to pass to :meth:`matplotlib.figure.Figure.savefig`, one of which is the
+``format`` argument. Currently sphinx-gallery supports PNG (default) and SVG
+output formats. To use SVG, you can do::
+
+    from sphinx_gallery.scrapers import matplotlib_scraper
+
+    class matplotlib_svg_scraper(object):
+
+        def __repr__(self):
+            return self.__class__.__name__
+
+        def __call__(self, *args, **kwargs):
+            return matplotlib_scraper(*args, format='svg', **kwargs)
```

Comment:
so the GIF scraper would be a little bit more complex, right? Because savefig probably works differently for GIFs than for static images?

Reply:
You need to figure out how to actually save a GIF, yes. This depends entirely upon what animation you're trying to extract. We could start with an easy case of MNE animated topomap if you want to. This would involve somehow finding the matplotlib `Animation` objects.

Reply:
I had actually found that animated GIFs were way easier with moviepy...though perhaps that'd add more complexity in the form of a new dependency we don't want...

Reply:
Optional dependency for optional functionality seems fine